modAionImpl/src/org/aion/zero/impl/db/AionRepositoryImpl.java

```diff
@@ -823,11 +823,7 @@ public void compactState() {
         ByteArrayKeyValueDatabase db = selectDatabase(dbType);
 
         Optional<byte[]> value = db.get(key);
-        if (value.isPresent()) {
-            return value.get();
-        } else {
-            return null;
-        }
+        return value.orElse(null);
```

Comment:
these changes are already applied in #842

Reply:
Merged #842 to `mpm` and rebased from `mpm`